buildSrc/src/main/groovy/com/google/firebase/gradle/plugins/FirebaseLibraryPlugin.java

```diff
@@ -54,6 +62,45 @@ public void apply(Project project) {
                 }
               });
     }
+    if (System.getenv().containsKey("METALAVA_BINARY_PATH")) {
+      String metalavaBinaryPath = System.getenv("METALAVA_BINARY_PATH");
+      AndroidSourceSet mainSourceSet = android.getSourceSets().getByName("main");
+      File apiTxtFile = new File(project.getProjectDir() + "/api.txt");
+      File baseLineFile = new File(project.getProjectDir() + "/baseline.txt");
+      File outputFile = new File(project.getRootProject().getBuildDir() + "/apiinfo/" + project.getPath().substring(1).replace(":", "_"));
+      File outputFileDir = outputFile.getParentFile();
+      if(!outputFileDir.exists()) {
```

Comment:
You should move it into the task action so that this does not happen if the task is not executed.